models/custom-application/src/custom-application/custom-application-draft/generator.ts

```diff
@@ -0,0 +1,23 @@
+import { fake, Generator } from '@commercetools-test-data/core';
+import { CustomApplicationMenuLinkDraft } from '../../custom-application-menu-link';
+import { CustomApplicationPermissionDraft } from '../../custom-application-permission';
+import { CustomApplicationSubmenuLinkDraft } from '../../custom-application-submenu-link';
+import type { TCustomApplicationDraft } from '../types';
+
+const generator = Generator<TCustomApplicationDraft>({
+  fields: {
+    url: fake((f) => f.internet.url()),
+    description: fake((f) => f.lorem.sentence()),
+    permissions: fake(() => [
+      CustomApplicationPermissionDraft.presets.ViewOnlyPermissions(),
+      CustomApplicationPermissionDraft.presets.ManageOnlyPermissions(),
+    ]),
+    icon: '<svg><path fill="#000000" /></svg>',
+    name: fake((f) => f.commerce.department()),
+    entryPointUriPath: fake((f) => f.lorem.word()),
```

Comment:
Shouldn't we "slugify" the entry point?

Reply:
yes, thanks [0fa991d](https://github.com/commercetools/test-data/pull/542/commits/0fa991dcfa46725c989b129e292800fba653ccd5)